test/weights.jl

```diff
@@ -96,6 +96,23 @@ end
     @test x == y
 end
 
+@testset "Unit weights" begin
+    wv = uweights(Float64, 3)
+    @test wv[1] === 1.
+    @test wv[1:3] == fill(1.0, 3)
+    @test wv[:] == fill(1.0, 3)
+    @test !isempty(wv)
+    @test length(wv) === 3
+    @test size(wv) === (3,)
+    @test sum(wv) === 3.
+    @test values(wv) == fill(1.0, 3)
+    @test varcorrection(wv) == 1/3
```

Comment:
```suggestion
    @test Base.varcorrection(wv) == 1/3
```
But it would also be nice to test `var`, just in case.

Reply:
It seems that this change broke the tests... I don't know if `varcorrection` is defined in base at all.

Reply:
Sorry, I mean StatsBase.